cmd/skywire-cli/commands/visor/shutdown.go

```diff
@@ -0,0 +1,22 @@
+package visor
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(
+		shutdownCmd,
+	)
+}
+
+var shutdownCmd = &cobra.Command{
+	Use:   "halt",
+	Short: "stop a running visor",
+	Run: func(_ *cobra.Command, args []string) {
+		rpcClient().Shutdown() //nolint
```

Comment:
We are not checking any return value here. We dont know if the service was actually shutdown.

Reply:
Nothing is returned if the visor is shut down!

If it did reply, it wouldnt be shut down.

The only way to really tell is to try it again or try some other rpc call.

It does actually work from my tests, but there is no good way to distinguish between the errors, i think